compose/cli/docopt_command.py

```diff
@@ -21,7 +22,12 @@ def __init__(self, command_class, options):
         self.options = options
 
     def parse(self, argv):
-        command_help = getdoc(self.command_class)
+        # Fix for http://bugs.python.org/issue12773
+        if hasattr(self.command_class, '__modified_doc__'):
+            command_help = cleandoc(self.command_class.__modified_doc__)
+        else:
+            command_help = getdoc(self.command_class)
+
```

Comment:
Was http://bugs.python.org/issue12773 not fixed before python 3.3 released? 

https://github.com/python/cpython/blob/3.3/Misc/NEWS#L4652 suggests it was, and testing against python 2.7.12, 3.3.5 and 3.5 locally there doesn't appear to be an issue in mutating the __doc__ string on user defined classes. So wondering if this part is really needed?